### ISSUE TITLE
Skip messages for certain period of time

### ISSUE THREAD

**Issue by marinkreso95** [spring-cloud/spring-cloud-stream-binder-aws-kinesis#128]
Is there a possibility to skip (not consume) messages for a certain period of time?

I have Amazon Kinesis stream that sends data every second but I want to consume message every 10 minutes, so out of 6000 messages I want to consume one message, not consume the rest, then wait another then minutes, consume the message and so on.

Is there some option for this in this library?

**Comment by artembilan** [comment]
No, there is no polling `MessageSource` for Kinesis at the moment: https://github.com/spring-projects/spring-integration-aws/issues/117

So, you are on your own to use a AWS Kinesis API directly in your application.

Then it is already out of scope of this project since you are not going to use any bindings specifics.
